lecture-notes/2-vcs.md

```diff
@@ -257,8 +260,7 @@ Changes to be committed:
 $
 ```
 
-This has now changed your `.git/` directory! Take a look. You should see
-something like this:
+After running this, you'll finally see a change in your `.git/` directory!
```

Comment:
I think "something like this" is important -- it may not be exact

Reply:
The hash should be exact, right? It's a blob, so its hash won't depend on date, author, or anything that would differ.

Reply:
might have different hash function or idk